### DISCUSSION TITLE
What does that string mean?

### DISCUSSION THREAD

**Discussion by Insoulum** [dyc3/opentogethertube#759] [Q&A] [answered]
https://github.com/dyc3/opentogethertube/blob/92a181f2d63e3f2b90ac86fcded231ef2bfe8fdf/client/src/locales/en.ts#L358
Hi! I would like to know in which case this message pops up? Can I have more details?

**Comment by dyc3** [comment, marked as answer]
In order to keep track of all the rooms, the state for rooms is kept loaded in the server's memory. To prevent the server from leaking memory, empty rooms are unloaded after a few minutes. Rooms can also be manually unloaded by an admin.

**Reply by Insoulum** [reply]
Thank you!
